profiles/killbill/src/main/java/org/killbill/billing/server/filters/RequestDataFilter.java

```diff
@@ -32,12 +34,13 @@
 @Singleton
 public class RequestDataFilter implements ContainerRequestFilter, ContainerResponseFilter {
 
+    private static final String REQUEST_ID_HEADER = "X-Request-Id";
 
```

Comment:
Should it be `X-Request-ID` or `X-Request-Id`. Googling shows me a lot of `X-Request-ID`. Unclear...